src/turndown.js

```diff
@@ -421,6 +421,174 @@ turndownService.addRule("div_with_class", {
   }
 })
 
+turndownService.addRule("multiple_choice_question", {
+  filter: (node, options) => {
+    if (node.nodeName === "DIV" && node.getAttribute("class")) {
+      if (
+        node.getAttribute("class") === "problem_question" &&
+        node.getElementsByClassName("problem_radio_input")
+      ) {
+        return true
+      }
+    }
+    return false
+  },
+  replacement: (content, node, options) => {
+    const questionId = node.getAttribute("id")
+    let questionMarkdown = Array.from(
+      node.getElementsByTagName("fieldset")[0].childNodes
+    )
+      .filter(child => {
+        if (
+          child.nodeName === "DIV" &&
+          child.getAttribute("class").includes("choice") &&
+          child.getElementsByClassName("problem_radio_input")
+        ) {
+          return true
+        }
+      })
+      .map(choice => {
+        const isCorrect =
+          choice
+            .getElementsByClassName("problem_radio_input")[0]
+            .getAttribute("correct") === "true"
+        return `{{<choice isCorrect="${isCorrect}">}}${turndownService.turndown(
+          choice.outerHTML
+        )}{{</choice>}}`
+      })
+      .join("\n")
+    questionMarkdown = `{{<choices>}}${questionMarkdown}{{</choices>}}`
+    const solution = helpers.getNextSibling(node, "div.problem_solution")
+
+    let solutionMarkdown = `{{<solution />}}`
+
+    if (solution) {
+      solutionMarkdown = `{{<solution>}}${turndownService.turndown(
+        solution.outerHTML
+      )}{{</solution>}}`
+    }
+
+    questionMarkdown = `${questionMarkdown}\n${solutionMarkdown}`
+
+    return `{{< multiple-choice-question questionId="${questionId}">}}${questionMarkdown}{{< /multiple-choice-question >}}`
+  }
+})
+
+turndownService.addRule("multiple_choice_questions_widget", {
+  filter: (node, options) => {
+    if (node.nodeName === "SCRIPT" && node.textContent.includes("quizMulti")) {
+      return true
+    }
+    return false
+  },
+  replacement: (content, node, options) => {
+    const jsParser = require("acorn")
+    const walk = require("acorn-walk")
+    const nodeText = node.textContent.replace(
+      "// There was an extra comma at the end of multiList array.",
```

Comment:
I was going to ask about this, then I searched through the parsed JSON and saw how often it appears. :joy:

Reply:
Yeah, it appears a lot, and also breaks the parsing because `textContent` gets rid of whitespace so everything after `//` gets interpreted as a comment. Not sure if there is a less hacky way to fix this, but this should be fine since there will definitely not be any new courses that use the `jQuizMe` library